.github/workflows/build-release.yml

```
@@ -3,7 +3,7 @@ name: '[CI] Build Release'
 on:
     push:
         tags:
-        -   '[0-9][0-9][0-9][0-9].[0-9][0-9].[0-9]'
+        -   '[0-9][0-9][0-9][0-9].[0-9][0-9].[0-9]+'
```

```suggestion
        -   '[0-9][0-9][0-9][0-9].[0-9][0-9].[0-9]+'  # YYYY.MM.MICRO
```

I suggest to add a comment, because not everyone is fluent with regexp.

Speaking of that, I think that written like this, the dot matches all characters. I.e., `2020X06Y1` would match?

---

It's closer to glob filtering so it should work as intended. See here:

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

---

+1 on the comment, and good call on the `.` . You need to escape it, yeah :)

---

Tried to push that example tag as a test and it didn't trigger the CI

https://github.com/pyscript/pyscript/releases/tag/2020X06Y1